include/node.h

```diff
@@ -201,6 +201,15 @@ class Node : public NodeBase<Tdim> {
   //! Apply velocity constraints
   void apply_velocity_constraints() override;
 
+  //! Assign absorbing constraint
+  //! \param[in] pwave_v P-wave velocity
```

Comment:
Hi, so I've been able to understand and make most of your other suggestions barring this one. How would I be able to access these values from the material properties? I had thought it would be through ```property_handle_``` but I'm currently getting an error about segmentation violation when I run ./mpmtest and it calls the function where I do this. Perhaps this is the right method but I need to initialize the properties somewhere. Thanks for your help